app/mailers/request_mailer.rb

```diff
@@ -244,8 +243,8 @@ def receive(email, raw_email, source = :mailin)
     # Find which info requests the email is for
     reply_info_requests = requests_matching_email(email)
 
-    # Nothing found, so save in holding pen
-    if reply_info_requests.empty?
+    # Nothing found OR multiple different info requests, so save in holding pen
+    if reply_info_requests.empty? or reply_info_requests.count > 1
```

Comment:
@gbp I've just thought of a potential issue. Due to the necessity to put messages with multiple BCC addresses in the holding pen (and I sort of thought that if there were multiple in Received, it should also do the same behaviour), I've simply added a check for multple info requests in general. Should this be the case, given that an array is expected, so I assume in general multiple info requests should be accepted? If not, I can change it to check on BCC addresses specifically.

(If I shouldn't change it though, I will fix the rubocop error to use `||` instead of `or`

Reply:
Not sure I follow the issue. Lets find a time to talk today